xarray/core/datatree.py

```diff
@@ -1546,7 +1546,11 @@ def to_netcdf(
             ``dask.delayed.Delayed`` object that can be computed later.
             Currently, ``compute=False`` is not supported.
         kwargs :
-            Addional keyword arguments to be passed to ``xarray.Dataset.to_netcdf``
+            Additional keyword arguments to be passed to ``xarray.Dataset.to_netcdf``
+
+        .. note::
+            Due to file format specifications the on-disk root group name
+            is always `/` overriding any given ``DataTree`` root node name.
```

Comment:
Do we double backticks for the slash instead of single ticks?

Reply:
Possibly? But if it doesn't work we can also fix it in #9033, which is the PR which will add these docstrings to the public API docs.

Reply:
I'm looking now to find out and I'll be sure to grab these changes in #9033

Reply:
And I can't tell because these docs aren't exposed yet in the build.
But in general the .rst will want two backticks for an inline literal. So maybe you want 
```
``"/"``` or ``/``
```
instead of
```
`/`
```
And leave 
```
``DataTree``
``` 
as is

Reply:
Thanks @flamingbear for looking into this. :+1: